controllers/dspipeline_controller.go

```diff
@@ -516,6 +516,38 @@ func (r *DSPAReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&rbacv1.Role{}).
 		Owns(&rbacv1.RoleBinding{}).
 		Owns(&routev1.Route{}).
+		// Watch for global ca bundle, if one is added to this namespace
+		// we need to reconcile on all the dspa's in this namespace
+		// so they may mount this cert in the appropriate containers
+		Watches(&source.Kind{Type: &corev1.ConfigMap{}},
```

Comment:
watching ConfigMap gobbles up memory doesn't it? Do we need to check for that and mitigate, possibly by increasing limits and or requests?

Reply:
We added a similar watcher for a pod, and didn't notice a major uptick (a lot of this is internally cached and heavily optimized in k8s I believe), and pod events work queue is likely to be insanely longer than the configmap events

Reply:
buut if it does cause issues, we will probably catch it in our perf testing and adjust accordingly